types/fakedoorcmd/message_test.go

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const expectedMsg = `The support for List hasn't been implemented yet.
+
+If you would like us to implement it, please drop by at:
+https://github.com/hazelcast/hazelcast-commandline-client/issues/48 and add a thumbs up 👍.
+We're happy to implement it quickly based on demand!
+`
+
+func TestNewFakeCommand(t *testing.T) {
```

Comment:
Would it make sense to convert this test to table driven test for other unimplemented DDSs as well?

Reply:
Sorry I don't see upside. I would be happy to discuss your insights though.

Reply:
Shouldn't we test that the message contains the correct data structure name and the GH link is correct?

Reply:
Hmm, I see. I think that would be a good test for root command. Let me add it and come back to this